apollo-api/src/commonMain/kotlin/com/apollographql/apollo/api/internal/json/ResponseJsonStreamReader.kt

```diff
@@ -104,7 +104,7 @@ open class ResponseJsonStreamReader(
     return when {
       isNextNull -> skipNext().let { null }
       isNextBoolean -> nextBoolean(false)
-      isNextNumber -> BigDecimal(nextString(false))
+      isNextNumber -> BigDecimal(nextString(false)!!)
```

Comment:
Is this something you really want to crash on if it's not working?

Reply:
It's not informative but `false` signals here that field is non optional, so we will never crash here but in other place where we resolve field value (inside `nextString` function)

Reply:
Shouldn't the value from that be non-nullable if that's the case?